README.md

```diff
@@ -120,8 +120,10 @@ the cluster was initially provisioned in the /credentials folder. Failure in doi
 ## For UPP Cluster
 ## LastPass: UPP - k8s Cluster Provisioning env variables
 
+## Build the docker image again so that the credentials gets baked into the docker image
```

Comment:
I have some worries about putting the credentials in the docker image as if someone forgets to build, he will update with the default credentials.
Also the credentials folders are now with credentials.{cluster-name}, which doesn't really match the "credentials" folder added by Dockerfile. 
I was going to suggest to not add them at all actually, and only mount them.